pyhidra/launcher.py

```diff
@@ -65,6 +66,25 @@ def _get_libjvm_path(java_home: Path) -> Path:
         if p.suffix != ".debuginfo":
             return p
 
+class _PyhidraImportLoader:
+    """ (internal) Finder hook for importlib to handle Python mod conflicts. """
+
+    def find_spec(self, name, path, target=None):
+
+        # If jvm is not started then there is nothing to find.
+        if not _jpype.isStarted():
+            return None
+
+        if name.endswith('_') and _jpype.isPackage(name[:-1]):
+            ms = _ModuleSpec(name, self)
+            ms._jname = name[:-1]
+            return ms
+
+    def create_module(self, spec):
+        return _jpype._JPackage(spec._jname)
```

Comment:
The need for `_jname` can be removed by having this just return `jpype.JPackage(spec.name[:-1]`.  This will also simplify the code in find_spec.